### ISSUE TITLE
NullReferenceException

### ISSUE THREAD

**Issue by AdriansRubins** [qwbarch/mirage#108]
Once round ends null reference exception on audio frame is triggered:
```bash
[Warning: Unity Log] Can not play a disabled audio source
[Warning: Unity Log] Can not play a disabled audio source
[Error  : Unity Log] NullReferenceException
Stack trace:
Mirage.Domain.Audio.Receiver.onReceiveFrame (Mirage.Domain.Audio.Frame+FrameData frameData, Microsoft.FSharp.Core.FSharpOption`1[T] _arg1) (at ./Mirage/Domain/Audio/Receiver.fs:66)
Mirage.Unity.AudioStream+onFrameRead@69.Invoke (Mirage.Domain.Audio.Frame+FrameData frameData) (at ./Mirage/Unity/AudioStream.fs:69)
Mirage.Domain.Audio.Sender+consumer@58-3.Invoke (Microsoft.FSharp.Core.FSharpOption`1[T] frameData) (at ./Mirage/Domain/Audio/Sender.fs:62)
Microsoft.FSharp.Control.AsyncPrimitives.CallThenInvokeNoHijackCheck[a,b] (Microsoft.FSharp.Control.AsyncActivation`1[T] ctxt, b result1, Microsoft.FSharp.Core.FSharpFunc`2[T,TResult] userCode) (at D:/a/_work/1/s/src/FSharp.Core/async.fs:528)
Microsoft.FSharp.Control.Trampoline.Execute (Microsoft.FSharp.Core.FSharpFunc`2[T,TResult] firstAction) (at D:/a/_work/1/s/src/FSharp.Core/async.fs:112)
--- End of stack trace from previous location where exception was thrown ---
<StartupCode$FSharp-Core>.$Async+StartImmediate@1798-1.Invoke (System.Runtime.ExceptionServices.ExceptionDispatchInfo edi) (at D:/a/_work/1/s/src/FSharp.Core/async.fs:1798)
Microsoft.FSharp.Control.AsyncPrimitives+StartWithContinuations@1188-2.Invoke (System.Runtime.ExceptionServices.ExceptionDispatchInfo x) (at <12b89e0ba184665de418f7f4c173580f>:IL_0000)
Microsoft.FSharp.Control.Trampoline.Execute (Microsoft.FSharp.Core.FSharpFunc`2[T,TResult] firstAction) (at D:/a/_work/1/s/src/FSharp.Core/async.fs:138)
<StartupCode$FSharp-Core>.$Async+clo@178-14.Invoke (System.Object o) (at D:/a/_work/1/s/src/FSharp.Core/async.fs:181)
UnityEngine.UnitySynchronizationContext+WorkRequest.Invoke () (at <e27997765c1848b09d8073e5d642717a>:IL_0002)
UnityEngine.UnitySynchronizationContext.Exec () (at <e27997765c1848b09d8073e5d642717a>:IL_005D)
UnityEngine.UnitySynchronizationContext.ExecuteTasks () (at <e27997765c1848b09d8073e5d642717a>:IL_0014)
```
![mirage-audio](https://github.com/user-attachments/assets/cc32d944-eaab-483c-9599-00f0a528c5e1)

Mods installed:

```
  name: BepInEx-BepInExPack
  name: anormaltwig-LateCompany
  name: FlipMods-TooManyEmotes
  name: Evaisa-HookGenPatcher
  name: Evaisa-LethalLib
  name: Evaisa-LethalThings
  name: Evaisa-FixPluginTypesSerialization
  name: MaxWasUnavailable-LethalModDataLib
  name: IAmBatby-LethalLevelLoader
  name: Rune580-LethalCompany_InputUtils
  name: XuXiaolan-ImmersiveScrap
  name: DrinkableWater-Brutal_Company_Minus
  name: qwbarch-NAudioLame
  name: Hamunii-DetourContext_Dispose_Fix
  name: Hamunii-AutoHookGenPatcher
  name: willis81808-LethalSettings
  name: Bobbie-NAudio
  name: qwbarch-MirageCore
  name: qwbarch-Mirage
  name: Lordfirespeed-OdinSerializer
  name: xilophor-LethalNetworkAPI
  name: zealsprince-Malfunctions
  name: Backrooms-Backrooms
  name: x753-Mimics
  name: AmesBoys-ImmortalSnail
  name: Coppertiel-MaskedEnemyOverhaulFork
  name: no00ob-LCSoundTool
  name: Clementinise-CustomSounds
  name: RugbugRedfern-Skinwalkers
  name: x753-Peepers
  name: SnoopCave-I_need_more_Bullets
  name: Enchanted_Games-SyncedSkinwalkers
  name: notnotnotswipez-MoreCompany
  name: rileyzzz-Skibidi_Toilet
  name: Midge-PushCompany
  name: WhiteSpike-Interactive_Terminal_API
  name: Sigurd-CSync
  name: malco-Lategame_Upgrades
  name: ButteryStancakes-EnemySoundFixes
  name: QuokkaCrew-MoreMonsters
  name: x753-More_Suits
  name: Verity-TooManySuits
  name: Badham_Mods-SCPFoundationDungeon
  name: jaspercreations-Scopophobia
  name: tinyhoot-ShipLoot
  name: sunnobunno-YippeeMod
  name: Ozone-Runtime_Netcode_Patcher
  name: KlutzyBubbles-BetterEmotes
  name: Drakorle-MoreItems
  name: FlipMods-LetMeLookDown
  name: ButteryStancakes-AccurateStaminaDisplay
  name: yeahimliam-Peter_Griffin_Suit
  name: doggosuki-Huntdown
  name: BudgetAirpods-InsanityRemastered
  name: SweetOnion-LethalSnap
  name: DiFFoZ-CompanyCruiserFix
  name: Kittenji-LaserPointerDetonator
  name: Piggy-JesterStompShake
  name: Electric131-IsThisTheWayICame
  name: MeatBallDemon-VileVendingMachine
  name: blink9803-AutoFlyToCompany
  name: TheWeavers-LethalBestiary
```

**Comment by qwbarch** [comment]
Thanks for the report, I'll look into this

**Comment by qwbarch** [comment]
Fixed in ``v1.13.0``
